sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_etl_sql_run_check_v1/query.sql

```diff
@@ -0,0 +1,4 @@
+SELECT
+  CURRENT_TIMESTAMP() AS last_run_timestamp,
+  @task_instance AS task_instance,
+  @run_id AS run_id
```

Comment:
Might be useful for debugging to know what time and Airflow task was last successful.

Reply:
Makes sense.  I'd suggest just calling it `run_timestamp` (corollary to `run_id`).